docs/csharp/roslyn-sdk/syntax-visualizer.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Explore code with the syntax visualizer
+description: The syntax visualizer provides a visual tool to explore the models the .NET Compiler Platform SDK generates for code.
+author: billwagner
+ms.author: wiwagn
+ms.date: 03/07/2018
+ms.topic: conceptual
+ms.prod: .net
+ms.devlang: devlang-csharp
+ms.custom: mvc
+---
+# Explore code with the syntax visualizer
+
+This article provides an overview of the Syntax Visualizer tool that ships as part of the .NET Compiler Platform ("Roslyn") SDK. The Syntax Visualizer is a tool window that helps you inspect and explore syntax trees. It's an essential tool to understand the models for code you want to analyze. It's also a debugging aid when you develop your own applications using the .NET Compiler Platform (“Roslyn”) SDK. Open this tool as you create your first analyzers. The visualizer helps you understand the models used by the APIs.
+
+[!INCLUDE[interactive-note](~/includes/roslyn-installation.md)]
```

Comment:
These instructions aren't correct. It tells you to choose the 'individual components' tab, but then it tells you to choose a workload. It should either say, choose the extension development workload (from the workloads tab) and then select the .NET complier platform SDK component in the summary pane on the right. Or it should say, choose the .NET compiler platform SDK component on the individual components tab (it's under the "Compilers, build tools, and runtimes" category). Sidenote: one could argue it is actually in the wrong category - perhaps it should be under the last category which is SDKs, libraries, and frameworks.

Reply:
I also updated the include to add the DGML editor, and removed the note about that in this article.